bcftools.wdl

```diff
@@ -165,31 +228,29 @@ task Sort {
     
 }
 
-task View {
+task Index {
     input {
         File inputFile
-        String outputPath = "output.vcf.gz"
+        Int minShift = 14
         String memory = "256M"
         Int timeMinutes = 1 + ceil(size(inputFile, "G"))
         String dockerImage = "quay.io/biocontainers/bcftools:1.10.2--h4f4756c_2"
-        String outputType = "z"
-        Int compressionLevel = 1
     }
 
     command {
         set -e
-        mkdir -p "$(dirname ~{outputPath})"
-        bcftools view \
-        -o ~{outputPath} \
-        -O ~{outputType} \
-        -l ~{compressionLevel} \
+        bcftools index \
+        --csi \
+        --tbi \
+        --force \
+        --min-shift ~{minShift} \
```

Comment:
Will this ever be used? There's no need to add obscure options unless they're actually useful. (No need to remove this now, just saying for future tasks)